bash_aliases.sh

```diff
@@ -1,26 +1,36 @@
-
+#!/bin/sh
 # Venv ALiases                                                                   
-alias venv-list='locate --regex "bin/activate"$'        # Locate /bin/activate paths for both conda and venv                         
-alias seder='sed  's/bin\/activate$//' '  # Cannot remember
+alias venv-list='locate --regex "bin/activate"$'    # Locate /bin/activate paths for both conda and venv                         
+alias seder='sed  's/bin\/activate$//' '            # Cannot remember
 
 #alias python=python2.7 
 
 # Conda Aliases                                                                  
-alias conda-remove-env='conda env remove --name $1'                             
-alias conda-create-env='conda create --name $1'                                 
-alias conda-lsp='conda list --name $1' # List conda packages                                          
-alias conda-ls='conda env list'        # List conda environments                                                  
-                                                                                 
-#Git Aliases                                                                     
-alias git-graph='git log --graph --oneline'                      # A simpler heuristic                                   
-alias git-show-lastn=' bash /home/meow/scripts/gitshowcom.sh $1' # Look behind at the nth commit back in the log.
+alias conda-create-env='conda env create --name'    # Creates a conda enviroment with the name as first argument($1)
+alias conda-remove-env='conda env remove --name'    # Removes a conda enviroment with the name as first argument($1)
+alias conda-eu="conda env update"                   # Updates the current conda environment
+alias conda-ls='conda env list'                     # List conda environments
+alias conda-lsp='conda list --name'                 # List conda packages                                          
+alias conda-i="conda install"                       # Installs a list of packages into a specified conda environment provided as arguments
+alias conda-r="conda remove"                        # Removes a list of packages from a specified conda environment provided as arguments
+
+#Git Aliases                                                
+
+alias git-graph="git log --graph --abbrev-commit --decorate --format=format:'%C(bold green)%h%C(reset) - %C(bold cyan)%aD%C(reset) %C(bold yellow)(%ar)%C(reset)%C(auto)%d%C(reset)%n''%C(white)%s%C(reset) %C(dim white)- %an%C(reset)' --all" # Simple and Beautiful way to display git log
+alias git-show-lastn=' bash $HOME/scripts/gitshowcom.sh' # Look behind at the nth commit back in the log.
 
 # Utils
-function lsize() { du -s ${1}/* | sort -rn | cut -f2- | xargs -d "\n" du -sh }
-alias ls-size='lsize $1wq'
+lsize() { 
```

Comment:
This seems too slow.

Reply:
It's getting stuck?

Reply:
![image](https://user-images.githubusercontent.com/42321810/193552801-5087e157-41d0-48fc-9973-489779f77fa4.png) 

Let's leave it like this lol

Reply:
It is quite slow indeed and requires optimization. It could be that running the command on a different mountpoint which doesnt allow reading is producing error and since I have disabled errors to show up... it ultimately shows up as stuck :/
https://0x0.st/oJXc.mp4

like here's a demo
Wanted to ask why do you use the du command twice?

Reply:
@justsaumit  actually I have no clue what's happending with the code lol. Let's not break what already worked lol. 

What terminal are you using? Is it Rust based?

Reply:
Like instead of writing this entire thing
```bash
du -s ./* | sort -rn | cut -f2- | xargs -d "\n" du -sh
```
you can use
```bash
  find . -mindepth 1 -maxdepth 1 | xargs -d "\n" du -sh | sort -hr
```
![image](https://user-images.githubusercontent.com/79687674/193711454-1e32847e-8ab7-420e-877b-a335ffb9c485.png)

#### Explanation: 
In 1st line I basically use a for loop to make 10 files each with around size of 'n' MB corresponding to their name. So 1.txt is about 1MB , 2.txt is about 2MB so on

2nd line is your original lsize function.

In 3rd line I try to minimize the usage of du and instead use find command.

_Funfact!_ you can just use ```bash 
ls -Sss1``` and get almost the same result
![image](https://user-images.githubusercontent.com/79687674/193711788-2d17a73c-786d-40c5-8b5a-71ec3db6b0fd.png)
Almost same because ```ls``` doesn't do well calculating the size of directories, hence you have to use du at some point.

### Benchmarking
Using time command to find which operation is more efficient and redirecting output to /dev/null so we just see the time command output.
![image](https://user-images.githubusercontent.com/79687674/193738812-6de3f817-e6f4-44c8-9722-0edb70de54dd.png)

Reply:
it's zsh 5.9 with [powerlevel10k](https://github.com/romkatv/powerlevel10k) as zsh theme/prompt. Pretty cool comes with git indication and all. Starship prompt is rust based i think

Reply:
If this works for directories let's merge it

Reply:
It does, however you do try 
```bash
find . -mindepth 1 -maxdepth 1 | xargs -d "\n" du -sh | sort -hr
``` 
on your own system and if it works and produces the same result. 
Then make sure to merge the pr :)

Reply:
Hmm, I like your method more since it shows the files as well as the sub-folders.